sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -826,6 +836,19 @@ func (host *nodeLanguageHost) InstallDependencies(
 
 	stdout.Write([]byte("Finished installing dependencies\n\n"))
 
+	if host.forceTsc {
+		// If we're forcing tsc for conformance testing this is our chance to run it before actually running the program.
+		// We probably want to see about making something like this an explicit "pulumi build" step, but for now shim'ing this
+		// in here works well enough for conformance testing.
+		tscCmd := exec.Command("npx", "tsc")
```

Comment:
We might want to run to `npx --package=typescript@3.8.3 tsc` so we ensure that the pu/pu types are compatible with that version.

Reply:
The package.json files have `"typescript": "^4.0.0"`, so I don't think 3.8.3 is going to be the one installed?

Reply:
You can force the version with `npx --package`, it will install the right version (into a cache folder, not the local node_modules) and run that instead.

I was mostly thinking that this would ensure that the types for pu/pu are 3.8.3 compatible, but maybe that is moot since we build pu/pu with 3.8.3, so it should aways be the case.